clientContext.ts

```diff
@@ -2,18 +2,21 @@ import React, {Context} from 'react';
 
 export interface ClientProps {
   nationalAvalancheCenterHost: string;
+  nationalAvalancheCenterWordpressHost: string;
   snowboundHost: string;
   nwacHost: string;
 }
 
 export const productionHosts = {
   nationalAvalancheCenterHost: 'https://api.avalanche.org',
+  nationalAvalancheCenterWordpressHost: 'https://devavycenters.wpengine.com', // TODO(skuznets): what's the production version here?
```

Comment:
https://avycenters.wpengine.com

This is likely it, but it does not (yet) host the API. I'll update this when Chris gets back to us.